src/explore-education-statistics-admin/src/services/publicationService.ts

```diff
@@ -131,7 +129,7 @@ const publicationService = {
         `/publication/${publicationId}/external-methodology`,
       )
       .catch(err => {
-        if (err.response.status !== 404) {
+        if (isAxiosError(err) && err?.response?.status !== 404) {
```

Comment:
Axios has its own `isAxiosError` function we could just use instead. We might want to just get rid of our implementation completely.